LibGAP.jl/src/conversion.jl

```diff
@@ -39,11 +40,12 @@ end
 
 from_gap(x,::Type{Any})    = x
 from_gap(x,::Type{String}) = from_gap_string( x )
+from_gap(x,::Type{Symbol}) = from_gap_symbol( x )
 from_gap(x,::Type{Bool})   = from_gap_bool( x )
 
 function from_gap_list_type( obj :: MPtr, element_type :: DataType ) 
     len_list = length(obj)
-    new_array = Array{element_type}(len_list)
+    new_array = Array{element_type,1}( undef, len_list)
```

Comment:
So this was broken all the time?

I notice that `from_gap_list_type` has zero code coverage; would be great to have at least one minimal test for one of the various callers to it.